toolkit/tools/pkg/imagecustomizerlib/testdata/verity-reinit-bootloader-reset.yaml

```diff
@@ -0,0 +1,21 @@
+previewFeatures:
+- reinitialize-verity
+
+os:
+  bootloader:
+    resetType: hard-reset
+
+  kernelCommandLine:
+    extraCommandLine:
+    - "rd.info"
+
+  additionalFiles:
+  - content: |
+      cat, dog, elephant, squirrel
+    destination: /usr/local/share/animals.txt
+
+  users:
+  - name: root
+    password:
```

Comment:
Should we remove this even from the test code?

Reply:
Yes, these were left over from manual testing.